superset/db_engine_specs/base.py

```diff
@@ -63,7 +63,8 @@
 from superset.databases.utils import get_table_metadata, make_url_safe
 from superset.errors import ErrorLevel, SupersetError, SupersetErrorType
 from superset.exceptions import DisallowedSQLFunction, OAuth2Error, OAuth2RedirectError
-from superset.sql_parse import ParsedQuery, SQLScript, Table
+from superset.sql.parse import SQLScript, Table
+from superset.sql_parse import ParsedQuery
```

Comment:
nit, but these two paths are really similar. What's the difference between them and will people be able to remember what goes where?

Reply:
Oh, sorry, I forgot to mention that the goal is to get rid of `sql_parse.py`.

The new `superset/sql/` directory will use only `sqlglot`, and as I'm working on SIP-117 I'm taking `sqlparse` logic from `sql_parse.py` and reimplementing it using `sqlglot` in `sql/parse/`.

Reply:
ok, perfect, that makes sense, then.